src/js/kangastus-wordpress.js

```diff
@@ -5,22 +5,43 @@
   'use strict';
   
   $.widget("custom.kangastusWordpress", {
-    
+
     options: {
       host: 'info-local.metatavu.io',
       port: 1234,
       secure: false
     },
-    
+
     _create : function() {
+      const protocol = this.options.secure ? 'https://' : 'http://';
+      this.wordpressUrl = `${protocol}${this.options.host}:${this.options.port}/wp-json/wp/v2/kangastus/`;
       this.updateQueue = [];
+      this.checkRemovedQueue = [];
     },
-    
+
     _resolveFileName(imageObject) {
        const fileType = imageObject['source_url'].split('.').pop();
        return `${imageObject.id}.${fileType}`;
     },
-    
+
+    _checkRemoved(id) {
+      return new Promise((resolve, reject) => {
+        $.ajax({
+          url: `${this.wordpressUrl}${id}`,
+          success: (data) => {
+            resolve(true);
```

Comment:
A bit confusing that _checkRemoved resolves to true when item has not been  removed